### PR TITLE
chore: enable more tests

### PR DESCRIPTION
**Pull request by peternhale**

### CODE REVIEW

File: package-scripts.js [hunk 1]
```diff
@@ -12,13 +12,13 @@ const path = require('path')
 sh.set('-e')
 
 setColors(['dim'])
-
-const testTypes = ['']
+process.env.TEST_SERIES = '1'
+const testTypes = ['', 'other']
```
Comment (mdonnalley) [inline package-scripts.js:16]: Does `unit` make more sense that `other` here?

Comment (peternhale) [reply package-scripts.js:16]: `other` is really not unit tests - these are the ones that hit aws.

Comment (mdonnalley) [reply package-scripts.js:16]: `integration`?

Comment (peternhale) [reply package-scripts.js:16]: I can do that

**Review by mdonnalley — APPROVED** [review]

**Pull request merged**